test/test_torchinductor.py

```diff
@@ -1093,6 +1096,15 @@ def fn(a, b):
             ),
         )
 
+    def test_gather2(self):
+        # 0d tensor
+        def fn(a, b):
+            return torch.gather(a, 0, b)
```

Comment:
Can you do something like `torch.gather(a, 0, b) + torch.gather(a, -1, b)` to test negative dim + 0d?